### DISCUSSION TITLE
“Bisq” is damaged and can’t be opened. You should move it to the Trash.

### DISCUSSION THREAD

**Discussion by badorahashem** [bisq-network/bisq#6492] [Q&A]
Description

After downloading and installing the Mac version, starting it fails with "“Bisq” is damaged and can’t be opened. You should move it to the Trash.

Version

1.9.8

Steps to reproduce

Download / verify Bisq package for Mac (v1.9.8)
Open dmg
In the newly opened window: copy Bisq application to the Applications folder
Try to open Bisq application (from the Applications folder)
Expected behavior

On the last step above, Bisq application should open.

Actual behavior

Opening Bisq fails with OSX error mentioned above.

Screenshots

<img width="372" alt="Screen Shot 2022-12-30 at 2 57 57 PM" src="https://user-images.githubusercontent.com/121675152/210072623-9f78f976-cb3e-4c00-8361-530e92f45df7.png">

Device or machine

macOS Monterey
version 12.6.2

**Comment by pazza83** [comment]
Hi @badorahashem I think there is an explanation and solution posted for this error on the Bisq wiki:

https://bisq.wiki/Downloading_and_installing#macOS
